JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -697,6 +718,38 @@ class AppInsightsTests extends TestClass {
             }
         });
 
+        this.testCase({
+            name: "AppInsights._onerror add document URL in case of CORS error",
+            test: () => {
+                // prepare
+                var sut = new Microsoft.ApplicationInsights.AppInsights(this.getAppInsightsSnippet());
```

Comment:
sut! Took it from Oleg's tests and wanted to ask him what that mean. But his authority made me stop and google it first, and it gave me "system under test". :)